Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevels.kt

```diff
@@ -0,0 +1,31 @@
+package de.rki.coronawarnapp.risk
+
+import com.google.android.gms.nearby.exposurenotification.ExposureSummary
+import de.rki.coronawarnapp.server.protocols.internal.AttenuationDurationOuterClass
+
+interface RiskLevels {
+
+    fun calculationNotPossibleBecauseNoKeys(): Boolean
+
+    fun calculationNotPossibleBecauseOfOutdatedResults(): Boolean
+
+    /**
+     * true if threshold is reached / if the duration of the activated tracing time is above the
+     * defined value
+     */
+    fun isActiveTracingTimeAboveThreshold(): Boolean
+
+    suspend fun isIncreasedRisk(): Boolean
```

Comment:
What do you think about refactoring this such that the `DefaultRiskLevels` implementation is not responsible for both retrieving the data and working on it?